src/domain/Lotto.js

```diff
@@ -0,0 +1,39 @@
+import Validator from "../utils/Validator";
+import LOTTO_SYSTEM from "../constants/lottoSystem";
+import { ERROR_MESSAGE } from "../constants/message";
+
+class Lotto {
+  _numbers;
+
+  constructor(numbers) {
+    if (!this.#isValidLottoNumbers(numbers))
+      throw new Error(ERROR_MESSAGE.invalidNumbers);
+
+    this._numbers = numbers;
+  }
+
+  get numbers() {
+    return [...this._numbers];
+  }
+
+  #isValidLottoNumbers(lottoNumbers) {
+    const lottoCount = LOTTO_SYSTEM.lottoDigitCount;
+    const lottoNumberType = "number";
+    const lottoMinNumber = LOTTO_SYSTEM.lottoRangeMinimum;
+    const lottoMaxNumber = LOTTO_SYSTEM.lottoRangeMaximum;
```

Comment:
이걸 한 번 더 변수에 할당하신건 가독성을 위한 부분일까요?
상수명과 변수명이 크게 다르지는 않으니 이렇게도 조금 더 간단하게 쓸 수 있을 것 같아요.

```suggestion
    const { lottoDigitCount, lottoRangeMinimum, lottoRangeMaximum } = LOTTO_SYSTEM
    const lottoNumberType = "number";
```

혹은 이것도 클래스 밖으로 아예 빼서 선언해도 될 것 같구요! 
(클래스 내부에 있어야만 하는 이유가 없긴 하니까요 😁 )